microsite/pages/en/on-demand.js

```diff
@@ -17,7 +17,7 @@ const ondemandDirectory = require('path').join(process.cwd(), 'data/on-demand');
 const ondemandMetadata = fs
   .readdirSync(ondemandDirectory)
   .map(file => yaml.load(fs.readFileSync(`./data/on-demand/${file}`, 'utf8')))
-  .sort((b, a) => a.date.toLowerCase().localeCompare(b.date.toLowerCase()));
+  .reverse();
```

Comment:
i'm not so sure that it's actually guaranteed that files are read in any sensible or even deterministic order - you probably need to add a `sort` before the `map`

Reply:
readdirSync follows the systems default sort, the filenames are made up of the dates in yyyymmdd. I reverse the whole thing to make sure the latest ones show up first. Happy to implement some other suggestion, I chose this approach to keep the date field in the file itself human friendly.

Reply:
I just don't trust what the default sort is :) especially across operating systems. Just moving the sort to be on the file names before reading their contents would make it guaranteed to be deterministic and also declares intent nicely.